R/render.R

```diff
@@ -235,7 +235,13 @@ infinite_moon_reader = function(moon, cast_from = '.') {
     rmarkdown::render(moon, envir = globalenv(), encoding = 'UTF-8')
   }
   build = local({
-    mtime = function() file.info(moon)[, 'mtime']
+
+    tmp_lines = rmarkdown:::read_lines_utf8(moon,"UTF-8")
+    tmp_css = rmarkdown:::parse_yaml_front_matter(tmp_lines)$output$`xaringan::moon_reader`$css
```

Comment:
The triple-colons `:::` will work for now because **xaringan** and **rmarkdown** happen to have the same maintainer (me). In general, we should avoid using `:::`.

Reply:
Glad to know I hit the right direction! I think I can replace those rmarkdown functions with some regex shenanigans but I'm happy you decided to take over the work!